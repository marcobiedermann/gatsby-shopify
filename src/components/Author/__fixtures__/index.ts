import { AuthorProps } from '..';

const author: AuthorProps = {
  name: 'John Doe',
};

export default author;
