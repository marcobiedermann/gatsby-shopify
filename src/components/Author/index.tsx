import React, { FC } from 'react';

export interface AuthorProps {
  name: string;
}

const Author: FC<AuthorProps> = (props) => {
  const { name } = props;

  return (
    <div>
      <span>{name}</span>
    </div>
  );
};

export default Author;
