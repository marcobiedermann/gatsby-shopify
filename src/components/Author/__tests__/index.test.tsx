import { render } from '@testing-library/react';
import React from 'react';
import Author from '..';
import author from '../__fixtures__';

describe('author component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Author {...author} />);

    expect(asFragment()).toMatchSnapshot();
  });
});
