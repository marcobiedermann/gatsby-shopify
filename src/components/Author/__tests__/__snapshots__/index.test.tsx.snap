// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`author component should render correctly 1`] = `
<DocumentFragment>
  <div>
    <span>
      John Doe
    </span>
  </div>
</DocumentFragment>
`;
